torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -416,5 +421,87 @@ GraphExecutorState ProfilingGraphExecutorImpl::getDebugState() {
   return state;
 }
 
+Node* createFallbackGraph(Block* b, ArrayRef<Value*> inputs, Graph* g) {
```

Comment:
Can you change the API to `replaceBlockWithFallBackGraph` ? I don't think there's ever going to be an API callsite where we don't want to do this